controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -463,3 +469,42 @@ func (r *DSCInitializationReconciler) watchAuthResource(ctx context.Context, a c
 
 	return nil
 }
+
+func (r *DSCInitializationReconciler) configureMonitoring(ctx context.Context, dsci *dsciv1.DSCInitialization) error {
+	// Create Monitoring CR singleton
+	defaultMonitoring := client.Object(&serviceApi.Monitoring{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       serviceApi.MonitoringKind,
+			APIVersion: serviceApi.GroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: serviceApi.MonitoringInstanceName,
+			OwnerReferences: []metav1.OwnerReference{{
```

Comment:
Since we set the owner reference here, shouln't the DSCI controller also own the Monitoring resource ?

Reply:
this is a good question.
i do not know what is the long term plan.
for me it loosk like not really need ownerref to DSCI, as montoring should be eventually able to run independently. as one day we should move the .spec.monitoring from DSCI to Monitoring.

Reply:
I don't know either, however since the spec is created out of the DSCI, it is either be owned or at least watched, otherwise, in case it gets deleted, then it won't get recreated.